mobile_verifier/src/heartbeats/mod.rs

```diff
@@ -107,6 +114,37 @@ pub enum OwnedKeyType {
     Wifi(PublicKeyBinary),
 }
 
+impl OwnedKeyType {
+    pub fn into_cbsd_id(self) -> Option<String> {
+        match self {
+            Self::Cbrs(cbsd_id) => Some(cbsd_id),
+            _ => None,
+        }
+    }
+}
+
+impl From<String> for OwnedKeyType {
+    fn from(s: String) -> Self {
+        Self::Cbrs(s)
+    }
+}
+
+impl From<PublicKeyBinary> for OwnedKeyType {
+    fn from(w: PublicKeyBinary) -> Self {
+        Self::Wifi(w)
+    }
+}
+
+impl PartialEq<KeyType<'_>> for OwnedKeyType {
+    fn eq(&self, rhs: &KeyType<'_>) -> bool {
+        match (self, rhs) {
+            (Self::Cbrs(lhs), KeyType::Cbrs(rhs)) => lhs == rhs,
```

Comment:
why do you need to deref the wifi keytype but not the cbrs keytype? b/c rust already implements PartialEq between Strings and &strs but `helium-crypto-rs` doesn't do the same between `PublicKeyBinary` and `&'a PublicKeyBinary`?

Reply:
yeah it's an impl thing